source/helpers/test/CaseDataConverter.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { replaceTagPart } from '../CaseDataConverter';
+
+describe('replaceTagPart', () => {
```

Comment:
This is my fault, but I haven't got the time to go thru testing with you yet. But when structuring tests you usually follow the Arrange, Act Assert syntax like this (notice the newlines):

```js
  it('replaces part at the beginning of the tag', () => {
    // The first part arranges the test (i.e. Arrange)
    const input = {
      tag: 'replace:example:test',
      part: 'replace',
      value: 'success'
    }
    const expectedOutput = 'success:example:test'
   
   // Here we run the test // Act
   const result = replaceTagPart(input.tag, input.part, input.value);

   // Check the result //Assert
   expect(result).toBe(output)
  });
```